documentation/modules/ROOT/pages/payloads-parquet.adoc

```diff
@@ -0,0 +1,564 @@
+////
+Copyright (c) 2024 Robert Bosch Manufacturing Solutions GmbH
+
+See the AUTHORS file(s) distributed with this work for additional information regarding authorship.
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/
+SPDX-License-Identifier: MPL-2.0
+////
+
+:page-partial:
+
+[[mapping-to-apache-parquet]]
+= Mapping to Apache Parquet
+
+== Overview
+
+Apache Parquet is an open source, column-oriented data file format designed for efficient data storage and retrieval.
+It provides high performance compression and encoding schemes to handle complex data in bulk and is supported in many programming languages and analytics tools.
+
+This document describes how Aspect Models are converted into Apache Parquet files, including the mapping of SAMM data types to Parquet types and the flattening strategy for nested structures.
+
+== Rules for the construction of Apache Parquet files matching an Aspect Model
```

Comment:
For long headings like this, please add a sensible short anchor, e.g.,

```suggestion
[[parquet-construction-rules]]
== Rules for the construction of Apache Parquet files matching an Aspect Model
```

The anchor is later used in the HTML to deep-link to this section, in this case ht<span>tps:</span>//eclipse-esmf.github.io/samm-specification/snapshot/payloads-parquet.html#parquet-construction-rules